google-analytics-chart.html

```diff
@@ -23,12 +26,28 @@
 @homepage https://googlewebcomponents.github.io/google-analytics
 -->
 
-<polymer-element
-  name="google-analytics-chart"
-  extends="google-analytics-query"
-  attributes="type width height options">
-
+<dom-module is="google-analytics-chart">
   <template>
+    <google-analytics-setup all-ready="{{setupReady}}"></google-analytics-setup>
+    <google-analytics-query id="query"
+      loading="{{loading}}"
+      get-data-response-handler="{{_boundResponseHandler}}"
+      data="{{data}}"
+      ids="{{ids}}"
+      start-date="{{startDate}}"
+      end-date="{{endDate}}"
+      metrics="{{metrics}}"
+      dimensions="{{dimensions}}"
+      sort="{{sort}}"
+      filters="{{filters}}"
+      segment="{{segment}}"
+      sampling-level="{{samplingLevel}}"
+      start-index="{{startIndex}}"
+      max-results="{{maxResults}}"
+      output="{{output}}"
+      fields="{{fields}}"
+      src-param="gwc-ga-chart"
```

Comment:
Please change this to `gwc-ga-query,gwc-ga-chart` so once I fix the other issue it will work.